projects.js

```diff
@@ -0,0 +1,114 @@
+      'A daily selection of privately personalized readsno accounts or sign ups required. has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    image: './assets/images/Img-Placeholder.svg',
+    technologies: ['css', 'html', 'bootstrap', 'ruby'],
+    livedemo: 'https://gordonotieno.github.io/portfolio-website/',
+    sourceCode: 'https://github.com/GordonOtieno/portfolio-website/',
+  },
+  {
+    name: 'Multi-Post Stories',
+    description:
+      'A daily selection of privately personalized readsno accounts or sign ups required. has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    image: './assets/images/Img-Placeholder.svg',
+    technologies: ['css', 'html', 'bootstrap', 'ruby'],
+    livedemo: 'https://gordonotieno.github.io/portfolio-website/',
+    sourceCode: 'https://github.com/GordonOtieno/portfolio-website/',
+  },
+  {
+    name: 'Multi-Post Stories',
+    description:
+      'A daily selection of privately personalized readsno accounts or sign ups required. has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    image: './assets/images/Img-Placeholder.svg',
+    technologies: ['css', 'html', 'bootstrap', 'ruby'],
+    livedemo: 'https://gordonotieno.github.io/portfolio-website/',
+    sourceCode: 'https://github.com/GordonOtieno/portfolio-website/',
+  },
+  {
+    name: 'Multi-Post Stories',
+    description:
+      'A daily selection of privately personalized readsno accounts or sign ups required. has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    image: './assets/images/Img-Placeholder.svg',
+    technologies: ['css', 'html', 'bootstrap', 'ruby'],
+    livedemo: 'https://gordonotieno.github.io/portfolio-website/',
+    sourceCode: 'https://github.com/GordonOtieno/portfolio-website/',
+  },
+];
+
+function getProjects(args) {
+  let html = '';
+  for (let i = 0; i < args.length; i += 1) {
+    html += ` <li class="project-list--li" tabindex="0">
+   <div class="project-detail">
+       <h3 class="project-title">${args[i].name}</h3>
+       <p class="project-desc">${args[i].description} </p>
+       <ul class="tech-list">
+           <li class="skills">${args[i].technologies[0]}</li>
+           <li class="skills">${args[i].technologies[1]}</li>
+           <li class="skills">${args[i].technologies[2]}</li>
+       </ul>
+   </div>
+       <button role="button" type="button" class="project_list--li__cta" data-toggle="modal" data-target="#myModal">See Project</button>
+</li>`;
+
+    document.querySelector('.work-section-container__project--list').innerHTML = html;
+  }
+}
+getProjects(projects);
+
+function displaymodal() {
+  let html = '';
+  html += ` <div class="modal fade" id="myModal" role="dialog">
+    <div class="modal-dialog">
+    
```

Comment:
Great job @GordonOtieno 🤝

- [ ] [Optional] You may kindly fix the background of your desktop popup. As you can see, you can easily scroll through the background of the main app. Please try to maintain only the popup on display. The background should not be scrollable. Try to make sure it is implemented the same way you did with the mobile popup.
![background](https://user-images.githubusercontent.com/63917725/180193225-131a6139-71b1-46da-b664-d4a5ff2732b9.PNG)